src/context.rs

```diff
@@ -1121,3 +1363,231 @@ pub fn read_resource_lines(path: &str) -> io::Result<Vec<String>> {
         .map(|s| s.to_string())
         .collect_vec())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // NOTE: if tests which use session object get added
```

Comment:
is this something we can leave in the code in working fashion ?
behind some flag ? env variable ?

Reply:
This is comment for situation which is absent here.
Just will minimize someone's spent time for investigation somewhere later.
Including me.

Reply:
o.k.

it seems like an option to have a test running with real scylla, and sound like a useful thing to keep in working manner ...

but I was miss reading the comment, i.e. those test doesn't really need it.

Reply:
Yes, tests that run using existing scylla will be really useful.
It is how it is done in scylla-rust-driver.

But it is definitely out of scope for this PR.

Reply:
o.k.